Shared/Samples/Show labels on layer/ShowLabelsOnLayerView.swift

```diff
@@ -0,0 +1,131 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ShowLabelsOnLayerView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    var body: some View {
+        // Create a map view to display the map.
+        MapView(map: model.map)
+            .alert(isPresented: $model.isShowingAlert, presentingError: model.alertError)
+    }
+}
+
+private extension ShowLabelsOnLayerView {
+    /// The view model for the sample.
+    class Model: ObservableObject {
```

Comment:
Since this type only have 1 ArcGIS source-of-truth type (`Map`), let's get rid of the view model and put everything in the view.